t/integration/test_canvas.py

```diff
@@ -1274,6 +1275,19 @@ def test_nested_chord_group_chain_group_tail(self, manager):
         res = sig.delay()
         assert res.get(timeout=TIMEOUT) == [[42, 42]]
 
+    @pytest.mark.xfail(TEST_BACKEND.startswith('redis://'), reason="Issue #6437")
+    def test_error_propagates_from_chord(self, manager):
+        try:
+            manager.app.backend.ensure_chords_allowed()
+        except NotImplementedError as e:
+            raise pytest.skip(e.args[0])
+
+        sig = add.s(1, 1) | fail.s() | group(add.s(1), add.s(1))
```

Comment:
should we also add si [immutable signature] in the test suit as well?

Reply:
I don't think it matters in this case.